src/uwtools/tests/drivers/test_driver.py

```diff
@@ -82,6 +100,153 @@ def schema():
 """.strip()
 
 
+def test_create_directory_structure(tmp_path):
```

Comment:
As the methods became more generic and moved up in the structure to the base class, the tests needed to be testing those base class methods directly.